qp/pdf.py

```diff
@@ -322,7 +323,7 @@ def gmm(x, *args):
             stdevs = popt[2]
         else:
             if self.samples is None:
-                self.samples = self.sample(using=using)
+                self.samples = self.sample(using=using, vb=vb)
```

Comment:
I think this change just causes `vb` from the call to `mix_mod_fit` to be propagated down.  It should solve the problem wherein performing the `mix_mod_fit` on a `qp.Ensemble` is significantly slowed due to printing all the samples, even when the call to `mix_mod_fit` has `vb=False`.